### PR TITLE
adding history and related apis

### PR DESCRIPTION
**Pull request by visualDust**

### CODE REVIEW

**Review by visualDust — COMMENTED** [review]
Should we merge before adding plotting apis, since this pr is supposed to add history db related things. Not sure, any idea¿

File: frontend/src/components/layout/AppFooter.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        display: "flex",
+        flexDirection: "column",
+        padding: "10px",
+        gap: "10px",
+      }}
+    >
+      <Divider />
+      <Logo styles={{ width: 50 }} withGlow={true} withLink={true} />
+      <Typography.Text type="tertiary" style={{ fontSize: 14 }}>
+        © 2023 - 2023 Neet Design. All rights reserved.
```
Comment (PommesPeter) [inline frontend/src/components/layout/AppFooter.tsx:20]:
```suggestion
        © 2023 - {new Date().getFullYear()} Neet Design. All rights reserved.
```

**Review by PommesPeter — COMMENTED** [review]
refine something

**Review by AndPuQing — APPROVED** [review]
May LGTM

**Review by PommesPeter — APPROVED** [review]
LGTM

**Pull request merged**